### PR TITLE
Fix Lifecycle routing

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #
https://github.com/meshery/meshery/issues/9655

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -278,6 +278,10 @@ ui-provider-build:
 ui-integration-tests: ui-setup
 	cd ui; npm run ci-test-integration; cd ..
 
+## rtk query code generation
+rtk-query-codegen:
```
Comment (leecalcote) [inline Makefile:282]: Will you put this in a different PR? This needs to be added to the list of dependency checks and we need to make sure that we’re actually publishing this package.

Comment (aabidsofi19) [reply Makefile]: got accidentaly pushed .

**Review by leecalcote — APPROVED** [review]
